frontend/src/components/Admin/AddUser.tsx

```diff
@@ -1,43 +1,27 @@
-import React from 'react';
-import { Card, Loader, Button, Form, Input, Select, TextArea } from 'semantic-ui-react';
+import React, { useState } from 'react';
+import { Card, Button, Form, Input, Select, TextArea } from 'semantic-ui-react';
+import ALL_ROLES from 'common-types/constants';
 import styles from './AddUser.module.css';
 import { Member, MembersAPI } from '../../API/MembersAPI';
 import ErrorModal from '../Modals/ErrorModal';
-import RolesAPI from '../../API/RolesAPI';
 import { getNetIDFromEmail, getRoleDescriptionFromRoleID, APICache, Emitters } from '../../utils';
```

Comment:
In the future, we might want to get rid of the `APICache`. It's not really being used in the first place and it's functionality is essentially being replaced by directly listening to changes in Firestore.